setup.py

```diff
@@ -243,7 +360,9 @@ def run(self):
 
     # Package info
     packages=find_packages(exclude=('test',)),
-
+    package_data={
+        package_name: ['*.lib', '*.dylib', '*.so']
```

Comment:
Is this necessary? We are able to package the `_C.so` without this.

Reply:
This is necessary to pack libpng/turbojpeg into the wheel distributions